LivingCity/benchmarking/runBenchmarks.py

```diff
@@ -182,20 +183,26 @@ def benchmark_one_run(number_of_run, benchmark_name, network_path):
 
     # ================ Metadata ================
     log("Saving metadata...")
-    metadata = {'elapsed_time_LivingCity': elapsed_time_LivingCity}
+    metadata = {'elapsed_time_simulation': elapsed_time_simulation}
     for output_line in full_output[2:].split("\\n"):
         if re.search('Number of blocks: '.format(component_name), output_line, flags=0):
             metadata['number_of_blocks'] = int(output_line.split()[-1])
         if re.search('Number of threads per block: '.format(component_name), output_line, flags=0):
             metadata['number_of_threads_per_block'] = int(output_line.split()[-1])
     
+    df_nodes = pd.read_csv(os.path.join(network_path, "nodes.csv"))
+    metadata['number_of_nodes'] = len(df_nodes)
+
+    df_edges = pd.read_csv(os.path.join(network_path, "edges.csv"))
```

Comment:
In the future, it would be great to make these parameters configurable (filename for edges).